File: in_Java/1009 Complement of Base 10 Integer.java
class Solution {
    public int bitwiseComplement(int N) {
        if(N == 0) return 1;
        int res = 0;
        int p = 0;
        int t;
        while(N != 0){
            t = N % 2;
            if(t == 0){
                res += Math.pow(2,p);
            }
            p++;
            N /= 2;
        }
        return res;
    }
}

File: in_Python/1009 Complement of Base 10 Integer.py
class Solution(object):
    def bitwiseComplement(self, N):
        """
        :type N: int
        :rtype: int
        """
        res = 0
        idx = 0
        if N == 0:
            return 1
        while N != 0:
            c = N % 2
            N /= 2
            res += (1 - c) * pow(2, idx)
            idx += 1
        return res
        